app/scripts/utils/fileExtention.js

```diff
@@ -0,0 +1,14 @@
+export default function fileExtention (name, defaultExt = '?') {
```

Comment:
i think a better solution would be to disable support for case insensetive filesystems